library/people/main.rb

```diff
@@ -0,0 +1,60 @@
+# rubocop:disable Metrics\CyclomaticComplexity, Metrics/MethodLength
+require './library/people/student'
+require './library/people/teacher'
+
+class PeopleIntializer
+  attr_reader :people
+
+  def initialize
+    @people = []
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'Your Library is empty, please add more people by clicking on 3'
+      return
+    end
+    @people.each do |person|
+      print "[#{person.class.name}] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age}\n"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+
+    option = gets.chomp
+
+    case option
+    when '1'
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      print 'Has parent permission? [Y/N]: '
+      permission_resp = gets.chomp
+      parent_permission = permission_resp.downcase == 'y'
+
+      student = Student.new(age, name, parent_permission)
+      @people << student
+
+      puts "Person created successfully\n"
+    when '2'
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      print 'Specialization: '
+      specialization = gets.chomp
+
+      teacher = Teacher.new(age, name, specialization)
```

Comment:
- [ ] Great job on refactoring your code 👏 however in this method try to follow best practices thought in [this solid lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), pay extra attention to the single responsibility rule. this method is printing, gathering information from the user and creating a new person based on choices made. try to cut this down into smaller steps

Reply:
Sorry, but I don't understand your request. I'm in pain, I got surgery this last weekend and I can't be sitting here all day. enlight me, please.